packages/react/src/themes/teams/components/Input/variants/Custom.ts

```diff
@@ -0,0 +1,16 @@
+export default {
```

Comment:
these temporary changes are introduced for the sake of example to reviewer - won't be merged